crates/ruff_linter/src/rules/flake8_pyi/rules/redundant_none_literal.rs

```diff
@@ -91,7 +96,16 @@ pub(crate) fn redundant_none_literal<'a>(checker: &mut Checker, literal_expr: &'
     // Provide a [`Fix`] when the complete `Literal` can be replaced. Applying the fix
     // can leave an unused import to be fixed by the `unused-import` rule.
     let fix = if other_literal_elements_seen {
-        None
+        create_fix_edit_2(checker.borrow_mut(), literal_expr, &literal_elements).map(|edit| {
```

Comment:
the type of the `checker` variable is already `&mut Checker` here, so this `borrow_mut()` call isn't necessary (and means we can avoid the import of the `BorrowMut` trait at the top of the file)

```suggestion
        create_fix_edit_2(checker, literal_expr, &literal_elements).map(|edit| {
```